spec/models/list_import/my_anime_list/row_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe ListImport::MyAnimeList::Row do
+  context 'with anime' do
+    let(:anime) { fixture('list_import/my_anime_list/toy-anime.json') }
```

Comment:
not sure if I like the way the fixture is being used. I think it might be better to just create your own object with the anime factory?

cc @NuckChorris